Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs

```diff
@@ -102,6 +103,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateHeight();
 		}
 
+		protected override void OnConfigurationChanged(Configuration newConfig)
+		{
+			base.OnConfigurationChanged(newConfig);
+
+			Element.Resources.Reload();
```

Comment:
Does it makes sense for all these reload calls to be tied to something else at the explat level? Like a themechanged event or something there on the Page itself?

Or instead of calling reload can it call into some other method on the Page itself?

Reply:
I didn't have the event before, but now that I do it makes more sense to do that more centrally, yes